core/tests/performance_framework/perf_services.py

```diff
@@ -68,6 +68,15 @@ def __init__(self, browser=DEFAULT_BROWSER_SOURCE):
             error_msg = 'Unsupported browser specified: %s' % browser
             raise ValueError(error_msg)
 
+    def load_url(self, page_url):
+        """Loads the specified url."""
```

Comment:
Yep, sounds like a good idea! If this results in client-side caching, it's worth mentioning. If it results in server-side caching, it's worth mentioning. It might also be worth mentioning that, because of the caching, future recorded metrics may be different than pre-caching (this is slightly obvious, but it doesn't hurt to say it, anyway).